tools/semgrep.yml

```diff
@@ -34,10 +34,8 @@ rules:
       - pattern-not: from zerver.models import filter_pattern_validator
       - pattern-not: from zerver.models import filter_format_validator
       - pattern-not: from zerver.models import generate_email_token_for_stream
-      - pattern-either:
-          - pattern: from zerver import $X
-          - pattern: from analytics import $X
-          - pattern: from confirmation import $X
+      - pattern: from $MODULE import $X
+      - pattern-where-python: "vars['$MODULE'] in ['zerver', 'analytics', 'confirmation']"
```

Comment:
This is likely also a big performance regression, but may not matter because the total number of imports in migrations is small.

Reply:
This adds 3s to total runtime so, I excluded this.